paasta_tools/check_marathon_services_replication.py

```diff
@@ -158,20 +158,25 @@ def check_smartstack_replication_for_instance(
     else:
         expected_count_per_location = int(expected_count / len(smartstack_replication_info))
         output = ''
+        output_critical = ''
+        output_ok = ''
         under_replication_per_location = []
 
         for location, available_backends in sorted(smartstack_replication_info.items()):
             num_available_in_location = available_backends.get(full_name, 0)
             under_replicated, ratio = is_under_replicated(
                 num_available_in_location, expected_count_per_location, crit_threshold)
             if under_replicated:
-                output += '- Service %s has %d out of %d expected instances in %s (CRITICAL: %d%%)\n' % (
+                output_critical += '- Service %s has %d out of %d expected instances in %s (CRITICAL: %d%%)\n' % (
                     full_name, num_available_in_location, expected_count_per_location, location, ratio)
             else:
-                output += '- Service %s has %d out of %d expected instances in %s (OK: %d%%)\n' % (
+                output_ok += '- Service %s has %d out of %d expected instances in %s (OK: %d%%)\n' % (
                     full_name, num_available_in_location, expected_count_per_location, location, ratio)
             under_replication_per_location.append(under_replicated)
 
+        output += output_critical
```

Comment:
how about add a newline to separate the two (if there is critical output to separate)